frontend/appflowy_flutter/lib/workspace/presentation/settings/widgets/theme_upload/theme_upload_decoration.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:dotted_border/dotted_border.dart';
+import 'package:flutter/material.dart';
+
+import 'theme_upload_view.dart';
+
+class ThemeUploadDecoration extends StatelessWidget {
```

Comment:
Would it be better if we moved this widget to flowy_infra_ui and made it a configurable widget? Then it could be used for theme upload and file upload in the database.

Reply:
I will hold off on this because I'd like to keep all of the theme upload widgets together. Some of the theme upload widgets use LocaleKeys which requires AppFlowy.